### DISCUSSION TITLE
How to remove black bars (top/bottom) on IOS app ?

### DISCUSSION THREAD

**Discussion by stephanberger4** [airsdk/Adobe-Runtime-Support#1587] [Q&A] [answered]
Hi, 

Before (with AIR 31 and 32) we needed to add different sizes of splash screens to our app ("`Default-375w-667h@2x~iphone`" , "`Default-414w-736h@3x~iphone`"...etc). 
I added 8 differents splash screen size
![image](https://user-images.githubusercontent.com/6382334/150436000-6b6a3f9a-562a-427b-8d59-46d288fa9058.png)
But, now, it doesn't seems to work anymore and everytime I test my app, it's not in fullscreen mode (black bar bottom and top). 

I've created an assets.car file ([online](http://www.applicationloader.net/appuploader/iosIcon.php)) but it seems to contain only the icons. 

Does someone know what to do to get rid of this black bars and make the app full screen ?

**Comment by marchbold** [comment, marked as answer]
You probably need a launch storyboard for your app and include the assets for it in your Assets.car 

https://docs.airnativeextensions.com/docs/tutorials/ios-icons-assets-car

**Reply by stephanberger4** [reply]
I did all of that... But maybe I did something wrong in on of the steps.. I'll try again and let you know

**Reply by marchbold** [reply]
Yeah I'd suggest just double checking, the Assets.car should contain the assets for your launch screen as well as the icons so you might be missing something there. And check that you have added the tag to your info additions.

**Reply by stephanberger4** [reply]
You were correct. I forgot to add `LaunchScreen.storyboardc` to my project ! 
Thank you !
